tests/TestCase/Panel/VariablesPanelTest.php

```diff
@@ -74,9 +74,9 @@ public function testShutdown()
 
         $unserializableDebugInfo = $this
             ->getMockBuilder('\stdClass')
-            ->setMethods(['__debugInfo'])
+            ->addMethods(['__debugInfo'])
             ->getMock();
-        $unserializableDebugInfo->expects($this->once())->method('__debugInfo')->willReturn([
+        $unserializableDebugInfo->expects($this->any())->method('__debugInfo')->willReturn([
```

Comment:
@markstory This test was failing as `__debugInfo()` was being called more than once. I couldn't figure out how/why that was happening. Most likely related to the Debugger internals changes.

Reply:
I suspect you're right. Having the mock always return the same value for __debugInfo should be fine.